components/elm/bld/namelist_files/use_cases/20thC_CMIP6_transient.xml

```diff
@@ -38,6 +38,7 @@
 <finidat hgrid="ne30np4" mask="oEC60to30v3">lnd/clm2/initdata_map/20180316.DECKv1b_A1.ne30_oEC.edison.clm2.r.1980-01-01-00000.8575c3f_c20190904.nc</finidat>
 
 <flanduse_timeseries hgrid="r05">lnd/clm2/surfdata_map/landuse.timeseries_0.5x0.5_hist_simyr1850-2015_c240308.nc</flanduse_timeseries>
+<flanduse_timeseries hgrid="r025" sim_year_range="1850-2015"  use_crop=".false." >lnd/clm2/surfdata_map/landuse.timeseries_0.25x0.25_hist_simyr1850-2015_c240125.nc</flanduse_timeseries>
```

Comment:
Hi @bishtgautam , it appears needing to remove `sim_year_range="1850-2015"` attribute above for F20TR.ne120pg2_r025_RRSwISC6to18E3r5 configuration to correctly pick up this file. Otherwise, build-namelist would complain no default flanduse_timeseries is set.

The test case has STARTDATE=1985-01-01. Not sure if it matters. Since 20TR (and this 20thC use case) is meant for the year range of 1850-2015. There should be no confusion taking away that attribute.